pkg/reaper/deployment_test.go

```diff
@@ -275,6 +276,32 @@ func TestLivenessProbe(t *testing.T) {
 	assert.Equal(t, expected, deployment.Spec.Template.Spec.Containers[0].LivenessProbe)
 }
 
+func TestEnableCors(t *testing.T) {
+	reaper := newTestReaper()
+
+	deployment := NewDeployment(reaper, newTestDatacenter(), nil, nil)
+	podSpec := deployment.Spec.Template.Spec
+	container := podSpec.Containers[0]
+	envVar := utils.FindEnvVar(container.Env, "REAPER_ENABLE_CROSS_ORIGIN")
+	assert.Nil(t, envVar)
+
+	reaper.Spec.EnableCors = false
```

Comment:
This is by default false (since it's not *bool), so testing the same thing as previous step. But it doesn't hurt.